### ISSUE TITLE
Hijack doesn't work with Emacs inferior ruby process?

### ISSUE THREAD

**Issue by bitops** [ileitch/hijack#15]
Hello!

I'm trying to use hijack as an inferior Ruby process in Emacs, but sending regions and definitions seems not to work. Code gets sent from my editing buffer to the repl, but the code doesn't actually seem to get eval'd.

Maybe an issue with how the hijack prompt is started?

**Comment by ileitch** [comment]
Does it work when not run from within Emacs?

**Comment by bitops** [comment]
@ileitch yeah, it definitely works. It also works from within Emacs when I just type stuff straight into the shell. What doesn't work is sending code from another buffer to the hijack repl. The 'send-region' command does some kind of eval magic which I don't fully understand, so that may be why. It _looks_ like the code gets sent and executed correctly, but then there's no output as I'd usually expect.
